frontend/Dockerfile.dev

```
@@ -0,0 +1,24 @@
+# Utiliser une version LTS spécifique de Node.js (branche 22.x), variante slim recommandée pour le développement
+FROM node:22.15.1-slim
+
+WORKDIR /app
+
+# Installer et configurer Corepack pour gérer PNPM
+RUN npm install -g corepack@latest
+RUN corepack enable
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)